### DISCUSSION TITLE
Add support for proxy list

### DISCUSSION THREAD

**Discussion by tboy1337** [masterking32/MasterHamsterKombatBot#66] [Ideas]
Could you add support for a proxy list and a setting in the config to only connect through proxy and dont connect otherwise?  It would help a lot if the bot could select a proxy from a text file and keep trying untill it finds one that works.

**Comment by AlexArGC** [comment]
It's not difficult to do. What to do with the rest of the settings?

**Comment by tboy1337** [comment]
What other settings?

**Reply by AlexArGC** [reply]
> What other settings?

If you start going through the list of free proxies, you will almost 100% get banned. For what? But with paid ones this is not necessary!

**Comment by tboy1337** [comment]
Hmm ok you might have a point there, could be ok to just use free proxys from a single country rather than jumping all over the planet?  Do you know any good cheap ones?  All the ones ive seen are quite expensive.
